modules/common/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/common/LoaderMessage.scala

```diff
@@ -166,7 +166,7 @@ object LoaderMessage {
 
   final case class Processor(artifact: String, version: Semver)
 
-  final case class Count(good: Long)
+  final case class Count(good: Long, bad: Option[Long])
```

Comment:
Do you also need to change the SchemaVer to 2-0-1 on [line 50](https://github.com/snowplow/snowplow-rdb-loader/blob/1db65cdad8f9a23b920d4bda4caa6c8418f45664/modules/common/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/common/LoaderMessage.scala#L50)?

Reply:
Good spot. Yep, it needs to be changed. I've missed it. I will create the new schema as well.

Reply:
@istreeter I've created [the PR for new schema](https://github.com/snowplow/iglu-central/pull/1248). It would be great if you can review it.